lua/snacks/notifier.lua

```diff
@@ -34,6 +34,7 @@ local uv = vim.uv or vim.loop
 ---@field style? snacks.notifier.style
 ---@field opts? fun(notif: snacks.notifier.Notif) -- dynamic opts
 ---@field hl? snacks.notifier.hl -- highlight overrides
+---@field hide_from_history? boolean
```

Comment:
Can yo change it to `history`? And then check for `opts.history ~= false`?

Reply:
nvim-notify already uses `hide_from_history`, should i check for both `history` and `hide_from_history`?

Reply:
nope, not needed to be compatible with `nvim-notify`